paddle/fluid/imperative/tracer.cc

```diff
@@ -12,13 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 #include "paddle/fluid/imperative/tracer.h"
+#include <iostream>
 #include <unordered_set>
 #include <utility>
 #include "paddle/fluid/platform/profiler.h"
-
 namespace paddle {
 namespace imperative {
 
+static std::shared_ptr<Tracer> current_tracer(nullptr);
```

Comment:
Try to use `g_` prefix for global variable.

Reply:
OK, thanks.